File: 0001_0100/0014/0014.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    a, b = (int(x) for x in f.read().split())
    m, n = a, b
    while n:
        m, n = n, m % n
    q.write(str(a // m * b))

File: 0001_0100/0014/0014.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var a, b int
	fmt.Fscan(f, &a, &b)
	m, n := a, b
	for n > 0 {
		m, n = n, m%n
	}
	fmt.Fprint(q, a/m*b)
}
